js_modules/dagster-ui/packages/ui-core/src/asset-data/AssetStaleStatusDataProvider.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 
+import {ApolloClient, gql, useApolloClient} from '../apollo-client';
```

Comment:
Not sure why these are getting reordered this way, but it's autofixed so I'm going with it.